library/geocore/src/main/java/it/geosolutions/imageio/maskband/DatasetLayoutImpl.java

```diff
@@ -0,0 +1,67 @@
+/*
+ *    ImageI/O-Ext - OpenSource Java Image translation Library
+ *    http://www.geo-solutions.it/
+ *    http://java.net/projects/imageio-ext/
+ *    (C) 2007 - 2015, GeoSolutions
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    either version 3 of the License, or (at your option) any later version.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package it.geosolutions.imageio.maskband;
+
+import java.io.File;
+
+/**
+ * Abstract implementation for {@link DatasetLayout} interface implementing default values for interface methods.
+ * 
+ * @author Nicola Lagomarsini GeoSolutions S.A.S.
+ */
+public abstract class DatasetLayoutImpl implements DatasetLayout {
+
```

Comment:
Is it this abstract class really needed? (It doesn't define any abstract method. maybe consider it a DefaultImpl?)